src/vs/editor/common/controller/oneCursor.ts

```diff
@@ -583,6 +586,63 @@ export class OneCursorOp {
 		return true;
 	}
 
+	public static moveSubWordLeft(cursor:OneCursor, inSelectionMode: boolean, ctx: IOneCursorOperationContext): boolean {
+		let position = cursor.getPosition();
+		var lineNumber = position.lineNumber;
```

Comment:
Use `const` as much as possible. If the value changes, like `positionFound` at L608, use `let`.